lib/iris/tests/__init__.py

```diff
@@ -702,13 +700,17 @@ def check_graphic(self, tol=None):
                     if err.errno != 17:
                         raise
 
-            figure.savefig(result_fname)
+            def save_figure_hash():
+                figure = plt.gcf()
+                figure.savefig(result_fname)
```

Comment:
I don't think this is a cosmetic change

previously the `figure = plt.gcf()` and `figure.savefig(result_fname)` were in the code, a little way apart.

i moved them together, without changing either line

minor point tho